www/include/monitoring/status/Services/service.php

```diff
@@ -528,6 +529,7 @@ function preInit() {
         _sid = '<?php echo $sid ?>';
         _tm = <?php echo $tM ?>;
         _o = '<?php echo $o; ?>';
+        _defaultStatusFilter = '<?php echo $defaultStatusFilter; ?>';
```

Comment:
why don't you use `<?=` instead of `<?php echo` ?

Reply:
because other (above) assignments were that way